scripts/ci/setup-dev-sso.sh

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env bash
+
+user_help () {
```

Comment:
minor, cosmetic: the other function definitions do not have a space before brackets

```suggestion
user_help() {
```